service_restaurant/src/main/kotlin/org/thechance/service_restaurant/api/usecases/UseCases.kt

```diff
@@ -0,0 +1,11 @@
+package org.thechance.service_restaurant.api.usecases
+
+import org.thechance.service_restaurant.entity.Restaurant
+
+interface GetRestaurantUseCase {
```

Comment:
It would be better to rename it getRestaurants instead of getRestaurant since it returs list of restaurants